### DISCUSSION TITLE
Add documentation regarding phone requirement

### DISCUSSION THREAD

**Discussion by maltfield** [d99kris/nchat#222] [Ideas]
This issue is an attempt to address the obvious question: does this `nchat` serve as an *alternative* to the WhatsApp app (ie: "is it a standalone desktop app?") or is it just yet-another-second-class-desktop-client that routes messages through the phone on the intranet?

### Solution

This is not a support request. This is a ticket to update the documentation. Please do *not* answer this question just on this GitHub issue.

The solution to this ticket should be to state, very clearly, in the `README.md` file if:

1. Using `nchat` still requires the user to first install and create an account in the official WhatsApp app, and all messages are routed through the phone (eg like whatsapp web), or
2. Using `nchat` still requires the user to first install and create an account in the official WhatsApp app, but afterwords WhatsApp can be uninstalled forever, and the desktop client can replace it (eg like Telegram), or
3. Using `nchat` is an entirely first-class desktop app. You can create an account in it directly, and you never need to use the official WhatsApp mobile app for any reason (eg like Wire)

Again, please add this information to the `README.md` file -- not to this ticket.

**Comment by d99kris** [comment]
I will move this to an enhancement / feature request.

**Reply by d99kris** [reply]
Since I may not have time to update documentation short-term, I can just share that for WhatsApp, a phone with the official app is needed for sign up and also keeping the account active, as far as I know. So basically your item 1.
